core/src/main/scala/dagr/core/tasksystem/NoOpInJvmTask.scala

```diff
@@ -26,6 +26,7 @@ package dagr.core.tasksystem
 /**
  * Trivial No-Op task that runs inside the JVM and does nothing.
  */
-class NoOpInJvmTask(name: String) extends SimpleInJvmTask {
+class NoOpInJvmTask(taskName: String) extends SimpleInJvmTask {
```

Comment:
Exactly - tried it, not allowed to override public vars, so sorry :disappointed: